core/domain/suggestion_services.py

```diff
@@ -1103,6 +1103,48 @@ def get_reviewable_translation_suggestions_by_offset(
     return translation_suggestions, next_offset
 
 
+def get_reviewable_translation_suggestion_target_ids(
+    user_id: str,
+    language_code: Optional[str] = None
+) -> List[str]:
+    """Returns a list of translation suggestions matching the
+    passed opportunity IDs which the user can review.
+
+    Args:
+        user_id: str. The ID of the user.
+        language_code: str|None. ISO 639-1 language code for which to filter.
+            If it is None, all available languages will be returned.
+
+    Returns:
+        list(str). A list of translation suggestion target ids
+        which the supplied user is permitted to review.
+    """
+    contribution_rights = user_services.get_user_contribution_rights(
+        user_id
+    )
+    language_codes = (
+        contribution_rights.can_review_translation_for_language_codes
+    )
+
+    # No language code means all languages.
```

Comment:
Could you please clarify here and in the docstring whether 'all / all available" languages only means the ones which the user has review rights for, or literally all languages that we offer?

Reply:
I replaced the comment with a variable definition.